tensorboard/plugins/mesh_visualizer/tf_mesh_dashboard/tf-mesh-dashboard.html

```diff
@@ -0,0 +1,279 @@
+                  Share viewpoint
+                </paper-radio-button>
+                <paper-tooltip animation-delay="0" for="user-radio-button" position="right" offset="0">
+                  Share viewpoint among all cameras.
+                </paper-tooltip>
+              </paper-radio-group>
+            </div>
+        </div>
+      </div>
+      <div class="center">
+        <template is="dom-if" if="[[_dataNotFound]]">
+          <div class="no-data-warning">
+            <h3>No point cloud data was found.</h3>
+            <p>Probable causes:</p>
+            <ul>
+              <li>You haven’t written any point cloud data to your event files.
+              <li>TensorBoard can’t find your event files.
+            </ul>
+            <p>
+            If you’re new to using TensorBoard, and want to find out how
+            to add data and set up your event files, check out the
+            <a href="https://github.com/tensorflow/tensorboard/blob/master/README.md">README</a>
+            and perhaps the <a href="https://www.tensorflow.org/get_started/summaries_and_tensorboard">TensorBoard tutorial</a>.
+            <p>
+            If you think TensorBoard is configured properly, please see
+            <a href="https://github.com/tensorflow/tensorboard/blob/master/README.md#my-tensorboard-isnt-showing-any-data-whats-wrong">the section of the README devoted to missing data problems</a>
+            and consider filing an issue on GitHub.
+          </div>
+        </template>
+        <template is="dom-if" if="[[!_dataNotFound]]">
+          <paper-input
+            no-label-float
+            label="Filter tags (regular expressions supported)"
+            value="{{_tagFilter}}"
+            class="search-input"
+          >
+          <iron-icon prefix icon="search"></iron-icon>
+          </paper-input>
+          <template is="dom-repeat" items="[[_categories]]" as="category">
+            <tf-category-pane category="[[category]]">
+              <tf-paginated-view
+                items="[[category.items]]"
+                pages="{{category._pages}}"
+              >
+                <template is="dom-repeat" items="[[category._pages]]" as="page">
+                  <template is="dom-if" if="[[page.active]]">
+                    <div class="layout horizontal wrap">
+                      <template is="dom-repeat" items="[[page.items]]">
+                        <tf-mesh-loader
+                          selected-view="[[_selectedView]]"
+                          run="[[item.run]]"
+                          tag="[[item.tag]]"
+                          sample="[[item.sample]]"
+                          of-samples="[[item.ofSamples]]"
+                          request-manager="[[_requestManager]]"
+                          class="tf-mesh-loader-container"
+                          on-camera-position-change="_onCameraPositionChanged"
+                          ></tf-mesh-loader>
```

Comment:
-2 indent

Reply:
not sure if I did it right, but formatting looks correct now